packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_inspector_web.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_visible_for_testing_member
```

Comment:
Why do we need this? That seems like a design concern.

Reply:
I defined some fields as `@visibleForTesting`, and the analyzer doesn't understand that this file is only used for testing (probably because it MAY not be used only for testing). The suggestion you gave above might allow me to remove this, however.

Reply:
(Done)